pkgs/development/python-modules/mmengine/default.nix

```diff
@@ -20,16 +21,16 @@
 
 buildPythonPackage rec {
   pname = "mmengine";
-  version = "0.10.1";
+  version = "0.10.3";
   format = "setuptools";
```

Comment:
Could you please replace this with `pyproject`?

Reply:
Done !